ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/MemoryCard/MemoryCard.tsx

```diff
@@ -0,0 +1,158 @@
+      </div>
+
+      <div className="overview-card__memory-tests u-flex--vertically">
+        <ul className="p-inline-list u-no-margin--bottom" data-test="tests">
+          {machine.memory_test_status.passed > 0 ? (
+            <li className="p-inline-list__item">
+              <Button
+                className="p-button--link"
+                element={Link}
+                to={testsTabUrl}
+                onClick={() =>
+                  sendAnalytics(
+                    "Machine details",
+                    "Memory tests passed link",
+                    "Machine summary tab"
+                  )
+                }
+              >
+                <Icon name={ICONS.success} />
+                {machine.memory_test_status.passed}
+              </Button>
+            </li>
+          ) : null}
+
+          {machine.memory_test_status.pending +
+            machine.memory_test_status.running >
+          0 ? (
+            <li className="p-inline-list__item">
+              <Button
+                className="p-button--link"
+                element={Link}
+                to={testsTabUrl}
+                onClick={() =>
+                  sendAnalytics(
+                    "Machine details",
+                    "Memory tests running link",
+                    "Machine summary tab"
+                  )
+                }
+              >
+                <Icon name={"pending"} />
+                {machine.memory_test_status.pending +
+                  machine.memory_test_status.running}
+              </Button>
+            </li>
+          ) : null}
+
+          {machine.memory_test_status.failed > 0 ? (
+            <li className="p-inline-list__item">
+              <Button
+                className="p-button--link"
+                element={Link}
+                to={testsTabUrl}
+                onClick={() =>
+                  sendAnalytics(
+                    "Machine details",
+                    "Memory tests failed",
+                    "Machine summary tab"
+                  )
+                }
+              >
+                <Icon name={ICONS.error} />
+                {machine.memory_test_status.failed}
+              </Button>
+            </li>
+          ) : null}
+
+          {hasTestsRun(machine, "memory") ? (
+            <li className="p-inline-list__item">
+              <Button
+                className="p-button--link"
+                element={Link}
+                to={testsTabUrl}
+                onClick={() =>
+                  sendAnalytics(
+                    "Machine details",
+                    "View memory tests results",
+                    "Machine summary tab"
+                  )
+                }
+              >
+                View results&nbsp;&rsaquo;
+              </Button>
+            </li>
+          ) : (
+            <li className="p-inline-list__item">
+              <span className="p-tooltip--top-left">
```

Comment:
Do you need these classes to wrap the tooltip component? Or is it doubling up the classes when rendered?

Reply:
Yeah it's definitely doubling up on classes. It doesn't actually cause any weirdness because the Tooltip component ends up higher up the DOM, but... it's unnecessary.

Reply:
thanks @anthonydillon good catch